parquet/src/test/java/org/apache/iceberg/avro/TestReadProjection.java

```diff
@@ -120,10 +121,9 @@ public void testEmptyProjection() throws Exception {
 
     Record projected = writeAndRead("empty_projection", schema, schema.select(), record);
 
-    Assert.assertNotNull("Should read a non-null record", projected);
+    Assertions.assertNotNull(projected, "Should read a non-null record");
     // this is expected because there are no values
-    Assertions.assertThatThrownBy(() -> projected.get(0))
```

Comment:
this was correct before, when migrating to JUnit5 we're also converting assertions to AssertJ